dart/test/stack_trace_test.dart

```diff
@@ -31,7 +31,7 @@ void main() {
       );
     });
 
-    test('cleanses absolute paths', () {
+    test('cleanes absolute paths', () {
```

Comment:
```suggestion
    test('cleans absolute paths', () {
```

I don't think `cleanes` exist either :D